deploy/ansible/ansible.cfg

```diff
@@ -91,7 +91,7 @@ hostfile		= inventory/default.ini
 roles_path		= roles:roles/local
 retry_files_save_path	= config/tmp/retry/
 log_path		= config/tmp/ansible.log
-vault_password_file	= config/vault_password
+#vault_password_file	= config/vault_password
```

Comment:
Maybe just remove?

Reply:
Left intentionally to show where Ansible expects to find this file in case we will ever be using encrypted vars.